src/Command.js

```diff
@@ -1,19 +1,23 @@
 import _ from "lodash";
 import dedent from "dedent";
+import log from "npmlog";
+
 import ChildProcessUtilities from "./ChildProcessUtilities";
 import FileSystemUtilities from "./FileSystemUtilities";
 import GitUtilities from "./GitUtilities";
-import ExitHandler from "./ExitHandler";
-import progressBar from "./progressBar";
-import Repository from "./Repository";
 import PackageUtilities from "./PackageUtilities";
-import logger from "./logger";
+import Repository from "./Repository";
+import filterFlags from "./utils/filterFlags";
+import writeLogFile from "./utils/writeLogFile";
+
+// handle log.success()
+log.addLevel("success", 3001, { fg: "green", bold: true });
 
 const DEFAULT_CONCURRENCY = 4;
 
 export const builder = {
   "loglevel": {
-    default: "info",
+    defaultDescription: "info",
```

Comment:
This is the `npmlog` default, and I didn't want to squash it during tests (where it is globally silenced).